package.json

```diff
@@ -695,7 +695,7 @@
         "portfinder": "^1.0.13",
         "request": "^2.83.0",
         "request-promise": "^4.2.2",
-        "vscode-azureappservice": "~0.16.2",
+        "vscode-azureappservice": "0.16.3",
```

Comment:
You can update the version on this, but leave the '~' in so that we automatically get bug fix updates